.github/workflows/figma.yml

```
@@ -0,0 +1,24 @@
+name: Figma
+
+on: workflow_dispatch
+
+jobs:
+  add-collection-to-figma:
+    # Prevents changesets action from creating a PR on forks
+    if: github.repository == 'RedHat-UX/red-hat-design-tokens'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Wdyt of packaging this up as a GitHub action?
```suggestion
      - uses: brianferry/figma-design-tokens@v1
```

---

Interesting thought, I haven't made my own action like that before so that would be fun to dive into.  I'll take a look

---

Started a repo for this action: https://github.com/brianferry/figma-create and added a workflow_dispatch action to this PR that would call it